### ISSUE TITLE
can't detect any bluetooth device

### ISSUE THREAD

**Issue by anusreepalangadan** [thon-ju/bluetooth_print#38]
D/BluetoothPrintPlugin( 9980): start scan 
D/BluetoothAdapter( 9980): STATE_ON
D/BluetoothLeScanner( 9980): onClientRegistered() - status=0 clientIf=6
I/flutter ( 9980): cur device status: 12
D/BluetoothAdapter( 9980): STATE_ON

repeating these lines while scanning

**Comment by turkialawlqy** [comment]
I had this problem and I solved it by activating the Location service at any time

**Comment by supanadit** [comment]
yeah its fixed the problem, but notes you must upgrade to version `3.0.1` in version `2.0.0` the problem remain still, even you have activate the location and the permission in androidmanifest.xml

**Comment by supanadit** [comment]
so think this issue can be closed now

**Comment by onnyprima** [comment]
> yeah `activating location service` its fixed the problem, but notes you must upgrade to version `3.0.1` in version `2.0.0` the problem remain still, even you have activate the location and the permission in androidmanifest.xml

still error "Stop Scan with callback" on android 11 activated location service

with this permission : 
<uses-permission android:name="android.permission.INTERNET" />
    <uses-permission android:name="android.permission.BLUETOOTH" /> 
    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" /> 
    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
